qiskit/providers/ibmq/api/rest/root.py

```diff
@@ -18,6 +18,8 @@
 
 from typing import Dict, List, Optional, Any
 
+from qiskit.providers.ibmq.apiconstants import ApiJobShareLevel
```

Comment:
Can you make this layer _not_ use `ApiJobShareLevel`, and instead just receive the strings? Is a bit of a judgement call, but in general we usually want to keep the `client/x.py` entities "smart-ish" rather than the rest layer.